tests/Vue/Pages/Results.spec.js

```diff
@@ -22,6 +22,25 @@ describe('Results.vue', () => {
         },
     }
 
+    it('displays intro text and instructions button', () => {
+        const wrapper = mount(Results, { mocks });
+
+        const intro = wrapper.find('#about-description');
+        expect(intro.isVisible()).toBe(true);
+
+        const instructionsButton = wrapper.find('.instructions-button');
+        expect(instructionsButton.isVisible()).toBe(true);
+
+    });
+
+    it('shows dialog after clicking the instructions button', async () => {
+        const wrapper = mount(Results, { mocks });
+        await wrapper.find('.instructions-button').trigger('click');
```

Comment:
Nice! In this case we are testing an interaction, which can be considered part of the "public interface" of the component (as clean as these things can be).